specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-11-01-preview/DynamicSummaries.json

```diff
@@ -0,0 +1,398 @@
+      "get": {
+        "x-ms-examples": {
+          "Get all summaries.": {
+            "$ref": "./examples/dynamicSummaries/GetSummaries.json"
+          }
+        },
+        "tags": [
+          "Summaries"
+        ],
+        "description": "Gets all summaries",
+        "operationId": "Summaries_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SummaryList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/dynamicSummaries/{summaryId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get a summary.": {
+            "$ref": "./examples/dynamicSummaries/GetSummaryById.json"
+          }
+        },
+        "tags": [
+          "Summaries"
+        ],
+        "description": "Gets a summary",
+        "operationId": "Summaries_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SummaryId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Summary"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
```

Comment:
>CloudError [](http://example.com/codeflow?start=67&length=10)

Does this inherit ../../../../../common-types/resource-management/v3/types.json ErrorResponse or similar?

Reply:
it doesn't inherit but it uses something similar "../../../common/2.0/types.json#/definitions/CloudErrorBody". following the way that Watchlist API (watchlists.json) defined the cloud error